articles/cognitive-services/Face/face-resource-container-config.md

```diff
@@ -43,7 +43,7 @@ The container reads user environment variables when the container is instantiate
 
 For example, the following commands use an environment variable to configure the console logging level to [LogLevel.Information](https://msdn.microsoft.com), then instantiates a container from the Face container image. The value of the environment variable overrides the default configuration setting.
 
-  ```Docker
+  ```DockerFile
```

Comment:
note: this still probably isn't a correct language block because this is actually a block of commands